koji/collection-mash-split.py

```diff
@@ -160,13 +161,15 @@ def inject_modulemd_yml(self, path, modulemd_yaml, modulemd_version=0, modulemd_
         cmd += " --queryformat='%{name}-%{epochnum}:%{version}-%{release}.%{arch}\n'"
         self.logger.debug("running %s" % cmd)
         status, output = kobo.shortcuts.run(cmd)
-        modules = yaml.safe_load(modulemd_yaml)
-        modules['data']['artifacts'] = {'rpms': output.splitlines()}
-        modules['data']['version'] = modulemd_version
-        modules['data']['context'] = modulemd_context
+        modules = list(yaml.safe_load_all(modulemd_yaml))
+        for module in modules:
+            modules['data']['version'] = modulemd_version
+            modules['data']['context'] = modulemd_context
+            if modules['data']['name'] == collection:
+                modules['data']['artifacts'] = {'rpms': output.splitlines()}
```

Comment:
Shouldn't this be `module` now?
```suggestion
            module['data']['version'] = modulemd_version
            module['data']['context'] = modulemd_context
            if module['data']['name'] == collection:
                module['data']['artifacts'] = {'rpms': output.splitlines()}
```

Reply:
of course!